ui/src/App.tsx

```diff
@@ -11,27 +12,20 @@ import TooltipPortal from 'src/portals/TooltipPortal'
 import NotesPortal from 'src/portals/NotesPortal'
 import Notifications from 'src/shared/components/notifications/Notifications'
 import OverlayController from 'src/overlays/components/OverlayController'
+import SetOrg from 'src/shared/containers/SetOrg'
 
 // Types
 import {AppState, CurrentPage, Theme} from 'src/types'
+import CreateOrgOverlay from './organizations/components/CreateOrgOverlay'
```

Comment:
route root and placement of import please :)

Reply:
darn auto imports!  thanks!